catalog/ui/search-ui/search-ui-app/src/main/resources/features.xml

```diff
@@ -70,11 +70,15 @@
         </configfile>
     </feature>
 
+    <feature name="simple-search-ui" install="auto" version="${project.version}">
```

Comment:
Right - they are independent features and should not depend on each other.  I broke them out so we could move closer to turning off the simple search ui.